runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -124,12 +124,23 @@ default boolean requestFocusOnNotification()
 	}
 
 	@ConfigItem(
-		keyName = "fontType",
+		keyName = "dynamicFontType",
```

Comment:
Change this effects existing users' configuration so I would probably leave it as is.